db/migrate/20181025230931_add_comments_to_vms_table.rb

```diff
@@ -0,0 +1,81 @@
+    change_column_comment :vms, :format, "The format of the VM's disk, such as VMDK."
+    change_column_comment :vms, :guid, "A value used to uniquely identify the VM internally."
+    change_column_comment :vms, :id, "The internal database ID."
+    change_column_comment :vms, :host_id, "ID of the host that the VM/Template resides on."
+    change_column_comment :vms, :last_perf_capture_on, "Timestamp for last performance metrics capture."
+    change_column_comment :vms, :last_scan_on, "Timestamp for last SmartState Analysis."
+    change_column_comment :vms, :last_scan_attempt_on, "Timestamp for last attempt to perform SmartState Analysis."
+    change_column_comment :vms, :last_sync_on, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :linked_clone, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :location, "The location or region of the VM."
+    change_column_comment :vms, :memory_hot_add_enabled, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :memory_hot_add_increment, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :memory_hot_add_limit, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :memory_limit, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :memory_reserve, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :memory_reserve_expand, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :memory_shares, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :memory_shares_level, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :miq_group_id, "MIQ Group associated with the VM."
+    change_column_comment :vms, :name, "The name of the VM/Template."
+    change_column_comment :vms, :orchestration_stack_id, "ID of the orchestration stack associated with the VM."
+    change_column_comment :vms, :power_state, "The current power state, typically 'on' or 'off'."
+    change_column_comment :vms, :previous_state, "The previous power state change."
+    change_column_comment :vms, :publicly_available, "Indicates whether the VM is public or private."
+    change_column_comment :vms, :raw_power_state, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :registered, "Indicates whether the VM registered to a Provider."
+    change_column_comment :vms, :resource_group_id, "ID of the resource group that the VM/Template belongs to."
+    change_column_comment :vms, :retired, "Indicates whether or not the VM is retired."
+    change_column_comment :vms, :retirement_last_warn, "Date of last retirement warning."
+    change_column_comment :vms, :retirement_requester, "Userid of the user who requested retirement."
+    change_column_comment :vms, :retirement_state, "The current retirement state."
+    change_column_comment :vms, :retirement_warn, "Warning message when VM was put into retirement."
+    change_column_comment :vms, :retires_on, "Date the VM is to be retired."
+    change_column_comment :vms, :smart, "[DEFINITION NEEDED]"
```

Comment:
I don't see this being set or used anywhere, @Fryguy do you know what this is?

Reply:
Hm, I want to say this had something to do with smart proxy. @gmcculloug I think we talked about this a long time ago, but my memory is hazy.

Reply:
That's correct, it is very old and no longer used.  It use to indicate that we added a file in with the other VM files to help identify the VM if it moved between systems.

Reply:
Sounds like we should drop the column